src/elm/Main/Pages/Msg.elm

```diff
@@ -39,6 +39,8 @@ import Pages.Org_.Repo_.Settings
 import Pages.Org_.Repo_.Tags
 
 
+{-| Msg : The main message type for the application, used to route messages to the correct page.
+-}
```

Comment:
i think this will get overwritten by `elm-land generate`, should we remove it?

Reply:
removed